build-scripts/gulp/gallery.js

```diff
@@ -19,32 +19,32 @@ require("./service-worker.js");
 require("./entry-html.js");
 require("./rollup.js");
 
-gulp.task("gather-gallery-demos", async function gatherDemos() {
-  const demoDir = path.resolve(paths.gallery_dir, "src/demos");
-  const files = glob.sync(path.resolve(demoDir, "**/*"));
+gulp.task("gather-gallery-pages", async function gatherPages() {
+  const pageDir = path.resolve(paths.gallery_dir, "src/pages");
+  const files = glob.sync(path.resolve(pageDir, "**/*"));
 
   const galleryBuild = path.resolve(paths.gallery_dir, "build");
   fs.mkdirSync(galleryBuild, { recursive: true });
 
-  let content = "export const DEMOS = {\n";
+  let content = "export const PAGES = {\n";
 
   const processed = new Set();
 
   for (const file of files) {
     if (fs.lstatSync(file).isDirectory()) {
       continue;
     }
-    demoId = file.substring(demoDir.length + 1, file.lastIndexOf("."));
+    const pageId = file.substring(pageDir.length + 1, file.lastIndexOf("."));
 
-    if (processed.has(demoId)) {
+    if (processed.has(pageId)) {
       continue;
     }
-    processed.add(demoId);
+    processed.add(pageId);
 
-    const [category, name] = demoId.split("/", 2);
+    const [category, name] = pageId.split("/", 2);
 
-    const demoFile = path.resolve(demoDir, `${demoId}.ts`);
-    const descriptionFile = path.resolve(demoDir, `${demoId}.markdown`);
+    const demoFile = path.resolve(pageDir, `${pageId}.ts`);
```

Comment:
```suggestion
    const pageFile = path.resolve(pageDir, `${pageId}.ts`);
```

Reply:
each page contains a demo and a description. The ts file is the demo.